checkout.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Adopt A Dog - Checkout</title>
+  </head>
+
+  <body style="background-color:powderblue;">
+
+<!-- This is the header on the page - in two sections for styling later.  -->
+    <header>
+      <article id="left-header">
+        <a href="index.html"><img src="images/adoptadog-logo.svg" alt="Logo for Adopt A Dog with silouette of dog."></a>
+        <img src="images/header-image.jpg" alt="Background image of brown dog.">
+        <h2 class="header-text">Seattle's premiere dog adoption agency</h2>
+      </article>
+
+      <article id="right-header">
+        <a href="checkout.html"><img src="images/cart-icon.svg" alt="Icon of a shopping cart.">$1234.56</a>
+        <p class="checkout"><a href="checkout.html">Checkout</a></p>
+      </article>
+    </header>
+
+<!--This is the navigation - current page has id of active for styling in navigation later-->
+    <ul>
+      <li class="navigation"><a href="index.html">Home /</a></li>
+      <li class="navigation"><a href="dogs.html">Dogs /</a></li>
+      <li class="navigation"><a href="blog.html">Blog</a></li>
+    </ul>
+
+<!-- Checkout form -->
+    <form method="GET">
+      <h1>Checkout</h1>
+
+<!-- Contact form elements -->
+      <fieldset class="contact">
+        <label>Name
+          <input type="text" name="contactName" required>
+        </label>
+
+        <label>Email Address
+          <input type="email" name="contactEmail" required>
+        </label>
+
+        <label>Street Address
+          <input type="text" name="contactStreet" required>
+        </label>
+
+        <label>City
+          <input type="text" name="contactCity" required>
+        </label>
+
+        <label>State
+          <select name="contactState" required>
+            <option value="">Please select</option>
+            <option value="WA">Washington</option>
+            <option value="OR">Oregon</option>
+            <option value="CA">California</option>
+          </select>
+        </label>
+
+        <label>ZIP Code
+          <input type="text" name="contactZIP" required>
+        </label>
+      </fieldset>
+
+<!-- Questions on form -->
+
+      <fieldset class="first-time">
+        <legend>First Time Adopter?</legend>
+        <label>
+          <input type="radio" name="firstTime" value="yes">
+          <span class="rad"></span>
```

Comment:
Not quite sure what these extra spans are for

Reply:
These were for styling the radio buttons later with CSS.

Reply:
gotcha, styling radio buttons is awful. I wouldn't worry about it too much, unless you really want.